src/main/java/edu/ucla/library/iiif/fester/handlers/GetStatusHandler.java

```diff
@@ -38,6 +40,9 @@ public void handle(final RoutingContext aContext) {
             final long usedMem = totalMem - freeMem;
             final double percentMem = (double) usedMem / (double) totalMem * 100D;
             final JsonObject memory = new JsonObject();
+            final String totalMemStr = totalMem + MB_STR;
```

Comment:
Perhaps use `concat()` for these three vars; or `StringBuffer`/`StringBuilder`, as those are more efficient in building strings from components?

Reply:
This isn't really necessary anymore. Newer versions of the JDK use StringBuilder under the hood in the JDK to optimize the concatenation.